tools/operator/launcher/k8s/connprofile.go

```diff
@@ -108,11 +111,11 @@ func (k K8s) GetK8sExternalIP(config networkspec.Config, serviceName string) (st
 }
 
 //GetK8sServicePort -- To get the port number of a fabric k8s component
-func (k K8s) GetK8sServicePort(serviceName, serviceType string, forHealth bool) (string, error) {
+func (k K8s) GetK8sServicePort(serviceName, serviceType, namespace string, forHealth bool) (string, error) {
```

Comment:
In Go, we don't put verbs in front of the function name for `Get` and `Set` like other languages, remove `Get` and give that the name of the object is K8s, we it provides the meaning for the function, so we don't need to include K8s on the function name, otherwise we are calling `K8s.K8sServicePort` the function should be ServicePort so the function call becomes `K8s.ServicePort()`

Reply:
yes agreed. renamed these functions to ServicePort.